### ISSUE TITLE
Translating ACF meta values

### ISSUE THREAD

**Issue by charlenejackie** [maximeschoeni/sublanguage#40]
Couldn't figure out how to translate Advabced Custom Fields values. When I switch the language through language tab in page editor, Gutenberg blocks do switch but the custom fields stay the same. I'm not able to enter values for the second language. Please help.

**Comment by maximeschoeni** [comment]
I think you need to enter the "language option" submenu of the post type tab, and check the corresponding meta key (not the one begining by "_")

**Comment by charlenejackie** [comment]
Actually I did check all the corresponding meta keys but still I cannot enter values for the second language.

**Comment by maximeschoeni** [comment]
Hello can you deinstall Sublanguage and confirm it's working without it?

**Comment by charlenejackie** [comment]
Sorry not sure if I understand. If I uninstall Sublanguage then I won't have the language switcher tabs in page editor, and it won't be possible for me to change the acf values per language then, which is what I'm trying to accomplish.

**Comment by maximeschoeni** [comment]
Sorry I asked you because ACF wasn't even working without Sublanguage on my test installation, but I retry on a clean install and its ok.
By the way did you uncheck the "Use asynchronous language switch" checkbox in the post meta language options?

**Comment by charlenejackie** [comment]
Oh I didn't sorry. I just unchecked it now and when I view the page editor in the default language all ACF fields show up, but they don't show up at all in the second language. I only see the field group title.

**Comment by maximeschoeni** [comment]
Strange... Did you try to change style to "seamless no metabox" instead of "Standard WP metabox" in the ACF field group settings?

**Comment by charlenejackie** [comment]
Yes, seamless no metabox is selected. Under Sublanguage Settings - Translate Post Types, do I need to check "Field Groups" and "Fields"? They were unchecked, I just checked them but nothing changed. What else do you think I should take a look at?

**Comment by maximeschoeni** [comment]
I don't think so... What is written on the title (when you hover on the checkbox label with the mouse and stay a bit).
Can you open the browser console and check if there is any javascript error?

**Comment by charlenejackie** [comment]
Well I just realized that I need to add the fields for the second language too! Haha. Not sure why I expected the existing fields to appear magically in the second language :) Sorry for taking your time.  You were such a huge help. I really really love this plugin. Thank you very much.
